auth_oidc/models/auth_oauth_provider.py

```diff
@@ -50,6 +50,46 @@ class AuthOauthProvider(models.Model):
         help="Additional parameters for the auth link. "
         "For example: {'prompt':'select_account'}"
     )
+    oidc_discovery_url = fields.Char(string="OIDC config URL")
+    supports_oidc_discovery = fields.Boolean(default=False)
+    auth_endpoint = fields.Char(required=False)
```

Comment:
Why making this field required=False, since it is mandatory in the discovery document?